app/dashboard/src/components/AriaComponents/Form/components/Field.tsx

```diff
@@ -20,8 +20,11 @@ import * as formContext from './useFormContext'
 export interface FieldComponentProps extends VariantProps<typeof FIELD_STYLES>, types.FieldProps {
   readonly 'data-testid'?: string | undefined
   readonly name: string
+  // We do not need to know the form fields.
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  readonly form?: types.FormInstance<any, any, any>
+  readonly form?: Omit<types.FormInstance<any>, 'handleSubmit'> & {
+    readonly handleSubmit?: (...args: never) => unknown
```

Comment:
Sorry but, Why?

Reply:
i guess the new `types.FormInstance<any>` is not equivalent to the old `types.FormInstance<any, any, any>` and so the type of `handleSubmit` becomes something that errors somewhere else

Reply:
```
Type 'FormInstance<Schema>' is not assignable to type 'FormInstance<any>'.
  Types of property 'handleSubmit' are incompatible.
    Type 'UseFormHandleSubmit<FieldValues<Schema>, TransformedValues<Schema>>' is not assignable to type 'UseFormHandleSubmit<any, any>'.
      Types of parameters 'onValid' and 'onValid' are incompatible.
        Type 'SubmitHandler<any>' is not assignable to type 'TransformedValues<Schema> extends undefined ? SubmitHandler<FieldValues<Schema>> : TransformedValues<Schema> extends FieldValues ? SubmitHandler<...> : never'.
```

Reply:
![CleanShot 2024-08-20 at 11 06 35@2x](https://github.com/user-attachments/assets/9883aaa0-f46a-46d6-87a0-6a90c0d7f098)

Well, making the `handleSubmit` compatible is painful: https://github.com/enso-org/enso/blob/597dd633432ff2cab493aa8f2d3744d89e4e5cc6/app/dashboard/src/components/AriaComponents/Form/Form.tsx#L110

=)

Reply:
broken link D:

Reply:
ok so i've switched `Field.tsx` to use generics and it seems (?) to work?